napari_aicsimageio/_settings_widget.py

```diff
@@ -0,0 +1,33 @@
+import napari
+from magicgui import magic_factory
+
+import napari_aicsimageio.core
+
+
+@magic_factory(
+    delimiter={"label": "Delimiter in scene labels:"},
+    in_mem_size={"label": "Threshold for out-of-memory loading (GB)"},
+    frac_mem_size={
+        "label": "Threshold for out-of-memory loading (<small>% free memory</small>):"
+    },
+    call_button="Apply Reader Settings and Close Widget",
```

Comment:
Does this _need_ to close the widget? I am tempted to keep it open the whole time :woman_shrugging:

Reply:
I have a small 13" screen, so Apply and close -> drag and drop -> scene selector was more comfortable.
Maybe I can add a checkbox for `Close on Apply`?

Reply:
Can there be two buttons next to each other? [Apply] <-- space --> [Apply and Close]

Reply:
eh. another checkbox is fine too

Reply:
I'll take a look 😊

Reply:
I don't like 2 buttons though. I think `Apply` and then people can close the widget manually is probably the most rational option.